!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  angular_x_Kd: 0.0
  angular_x_Ki: 0.0
  angular_x_Kp: 0.0
  angular_y_Kd: 0.0
  angular_y_Ki: 0.0
  angular_y_Kp: 0.0
  angular_z_Kd: 280.0
  angular_z_Ki: 0.0
  angular_z_Kp: 20.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      angular_x_Kd: 0.0
      angular_x_Ki: 0.0
      angular_x_Kp: 0.0
      angular_y_Kd: 0.0
      angular_y_Ki: 0.0
      angular_y_Kp: 0.0
      angular_z_Kd: 280.0
      angular_z_Ki: 0.0
      angular_z_Kp: 20.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      id: 0
      linear_x_Kd: 70.0
      linear_x_Ki: 10.0
      linear_x_Kp: 100.0
      linear_y_Kd: 70.0
      linear_y_Ki: 10.0
      linear_y_Kp: 100.0
      linear_z_Kd: 0.0
      linear_z_Ki: 0.0
      linear_z_Kp: 0.0
      name: Default
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      state: true
      type: ''
    state: []
  linear_x_Kd: 70.0
  linear_x_Ki: 10.0
  linear_x_Kp: 100.0
  linear_y_Kd: 70.0
  linear_y_Ki: 10.0
  linear_y_Kp: 100.0
  linear_z_Kd: 0.0
  linear_z_Ki: 0.0
  linear_z_Kp: 0.0
state: []
